src/Beacon/BeaconDemo/Bus.cs

```diff
@@ -1,9 +1,9 @@
-﻿using System;
+﻿using NetMQ;
+using NetMQ.Sockets;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Net;
-using NetMQ;
-using NetMQ.Sockets;
```

Comment:
It's easier to review a huge PR like this if you keep unrelated refactorings separate.